src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,41 @@
+
+[Telemetry in context](https://docswebsitedevelop-doc7761logsintro.gtsb.io/docs/logs/logs-everywhere/telemetry-in-context/) for the Ruby agent connects your logs with all of your telemetry data for your apps, hosts, and other entities. Bringing all of this data together in a single tool helps you quickly:
+
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+For example, with telemetry in context, you can easily see [log patterns](/docs/logs/ui-data/find-unusual-logs-log-patterns), or show [surrounding logs](/docs/logs/troubleshooting/find-issues-cause-or-impact-surrounding-logs) at the time an error occurred in your app.
+
+## Get started [#enable]
+
+To enable telemetry in context for APM apps monitored by Ruby, install or update to [Ruby agent version 8.6.0  or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes). **REVIEWER: version tbd**
+
+That's it! To make the most of your logging data:
+
+* Explore the logging data across your platform with our [Logs UI](/docs/logs/log-management/ui-data/use-logs-ui/).
+* See your logs in context of your app's performance in the [APM UI](/docs/logs/log-management/ui-data/use-logs-ui/#links). Troubleshoot [errors](/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems/) with [distributed tracing](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data), stack traces, application logs, and more.
+* Get deeper visibility into both your application and your platform performance data by forwarding your logs with our [infrastructure monitoring agent](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/). Review your [infrastructure logs](/docs/logs/log-management/ui-data/use-logs-ui/#links) in the UI.
+* Set up [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/).
+* [Query your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/) and [create dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+
+**REVIEWER: These bullets come from the original Ruby logs in context doc. Will need to be updated.**
+
+## Optional configuration [#config]
+
+When you update to telemetry in context, three changes are made to your configuration file:
+
+* Application log forwarding enabled by default
+* Maximum log sample stored (10,000 lines by default)
+* Logging metrics, including `trace.id`, `span.id`, `entity_guid`, and `hostname`
```

Comment:
This isn't really about configuration -- this is the effect that we have by doing telemetry-in-context. The attributes get applied to log lines, though, not the metrics (but not sure this belongs here).

```suggestion
* Logging metrics, including `trace.id`, `span.id`, `entity.guid`, and `hostname`
```

Reply:
Clearly I got things mixed up. Will need your help to reword!